src/main/scala/firrtl/backends/experimental/rtlil/RtlilEmitter.scala

```diff
@@ -0,0 +1,888 @@
+package firrtl.backends.experimental.rtlil
+
+import java.io.Writer
+import firrtl._
+import firrtl.PrimOps._
+import firrtl.ir._
+import firrtl.Utils._
+import firrtl.WrappedExpression._
+import firrtl.traversals.Foreachers._
+import firrtl.annotations._
+import firrtl.options.Dependency
+import firrtl.passes.LowerTypes
+import firrtl.passes.MemPortUtils.memPortField
+import firrtl.stage.TransformManager
+
+import scala.collection.mutable
+import scala.collection.mutable.ArrayBuffer
+import scala.language.postfixOps
+
+
+object RtlilEmitter extends SeqTransform with Emitter with DependencyAPIMigration {
```

Comment:
Since this is experimental, please make the `RtlilEmitter` object `private [firrtl]`.